src/operation/decision_model.jl

```diff
@@ -318,16 +319,20 @@ function build_problem!(model::DecisionModel)
 end
 
 function reset!(model::OperationModel)
-    if built_for_recurrent_solves(model)
+    # TODO-PJ: This is needed until we remove the ParameterJuMP dependency
+    # TODO: If the optimization container already has initial condition data, do not re-initialize
```

Comment:
@daniel-thom see here to consider one more case

Reply:
Is this comment resolved by line 335?

Reply:
No, this just resets the ic_container. If I run build again on the same problem it will try to intialize again since in 328 we are re-setting the optimization container.